src/Components/CreateImageWizard/CreateImageWizard.js

```diff
@@ -501,20 +503,21 @@ const formStepHistory = (composeRequest) => {
 };
 
 const CreateImageWizard = () => {
-  const awsTarget = insights.chrome.isBeta() ? awsTargetBeta : awsTargetStable;
-
+  const chrome = useChrome();
   const dispatch = useDispatch();
   const navigate = useNavigate();
   const location = useLocation();
 
+  const awsTarget = chrome.isBeta() ? awsTargetBeta : awsTargetStable;
```

Comment:
We just merged https://github.com/RedHatInsights/image-builder-frontend/pull/1009 which adds an isBeta() helper function... the helper function is useful because it also exposes these features in the qa environment. Let's use it here! :)

Reply:
Totally, great helper! :)
It's bit of a shame it can't use the `useChrome` hook as frontend team tries to push on removing the dom `chrome`, but I guess that won't happen anytime soon :+1:

Reply:
Yeah, I got the same feeling, shame, but maybe there will be a solution out there for us soon (might be good to bring it up with the frontend team, they were very helpful whenever I approached them)